dlt/destinations/path_utils.py

```diff
@@ -139,7 +139,10 @@ def prepare_params(
     if job_info:
         table_name = job_info.table_name
         file_id = job_info.file_id
-        ext = job_info.file_format
+        if job_info.has_compression_ext:
```

Comment:
more or less same as above, maybe ParsedLoadJobFileName could have a method "full_extension" which does this little code block (and explains what it does)